### PR TITLE
[feature #157801300] Czar should be able to draw card to populate que…

### PR DESCRIPTION
**Pull request by clintfidel**
#### What does this PR do?
- This PR ensures that only czar is able to click black card to draw question
#### Description of Task to be completed?
- Czar should be able to click black card to draw question

#### How should this be manually tested?
- visit application url [here](https://asgard-cfh-staging.herokuapp.com) and play game, once you are the czar click black card to draw questions 

#### What are the relevant pivotal tracker stories?
[157801300](https://www.pivotaltracker.com/story/show/157801300)

### Screen Shots
screen shot 2018-06-21 at 1 56 37 pm

### CODE REVIEW

File: public/css/style.css [hunk 1]
```diff
@@ -557,8 +557,65 @@ carousel-indicators .active {
     height: 100%;
     width: 100%;
     cursor: pointer;
+    object-fit: cover;
+    border-radius: 7px;
+    /* border: 2px solid transparent; */
```
Comment (fortunee) [inline public/css/style.css]: @clintfidel please get rid of the commented code

File: public/css/style.css [hunk 2]
```diff
@@ -557,8 +557,65 @@ carousel-indicators .active {
...
+  /* entire container, keeps perspective */
+.flip-container {
+  perspective: 1000px;
+  height: 100%;
+  width: 100%
+}
+	/* flip the pane when hovered */
+	.flip-container .flipper, .flip-container.hover .flipper {
+		transform: rotateY(180deg);
+	}
```
Comment (fortunee) [inline public/css/style.css]: Use proper indentation here

File: public/css/style.css [hunk 3]
```diff
@@ -557,8 +557,65 @@ carousel-indicators .active {
...
+
+/* .flip-container, .front, .back {
+  height: 100%;
+  width: 100%;
+} */
```
Comment (fortunee) [inline public/css/style.css]: Remove the commented out code here

**Review by fortunee — COMMENTED** [review]
Hi, @clintfidel please review my comments and ensure you get rid of all commented out code. I'll merge as soon as you're done. Thanks 😄

**Comment by clintfidel** [comment]
Hi, @fortunee feedback implemented.

**Pull request merged**